packages/cli/src/dbt/manifest.ts

```diff
@@ -12,7 +12,7 @@ export const loadManifest = async ({ targetDir }: LoadManifestArgs) => {
             await fs.readFile(filename, { encoding: 'utf-8' }),
         ) as any;
         return manifest;
-    } catch (err) {
+    } catch (err: any) {
```

Comment:
why was this needed?

Reply:
I got these errors when building 

![image](https://user-images.githubusercontent.com/1983672/170022283-5b85faec-c213-4e24-a59a-5a1180352c43.png)

Reply:
In one of the latest Typescript releases they changed the error type from `any` to `unknown`.  Do you have a different version installed locally or have we changed the version recently ? 🤔

Reply:
It is part of 4.4.0 [(release notes)](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-4.html#using-unknown-in-catch-variables) and we use `"typescript": "^4.2.3"` for the cli package but `"typescript": "^4.5.5",` for the frontend package so your IDE might be using the latest version for all the project ignoring the version at the workspace level

Reply:
I installed this version manually while I was trying to fix the conflicts on CLI. I resintalled all node_modules and I got  typescript: '^4.2.3' version now, now the `unknown` doesn't happen. 

Still, this is not a breaking change, so I'd suggest to leave it as it is